__tests__/authn-api.test.ts

```diff
@@ -0,0 +1,15 @@
+import { dataSources, mockResponse, mockRequestUrl } from './setupJest';
+
+describe('AuthnAPI', () => {
+    let AuthnAPI = dataSources.authn;
+
+    describe('login', () => {
+        it('queries the expected URL with credentials', async () => {
+            mockResponse({"ok": true});
+            const result = await AuthnAPI.login('username', 'password');
+            expect(mockRequestUrl()).toContainRequestPath('/authn/login');
+            expect(result).toEqual({ ok: true });
```

Comment:
Is this just testing that the response is what we mocked it to be?  If so, is this test actually testing any code?

Reply:
Thanks for looking Justin. I had the same thought at first but ultimately since the methods can pick different parts of the FOLIO response for returning, it didn't seem totally pointless. Example, FOLIO returns `{ holdings: [{..1..}, {...2...}] }` but our API method takes that response and only returns the array inside `holdings` without the key. There are different scenarios across methods that do different things with the FOLIO responses.